docs/editors/overview.md

```diff
@@ -4,6 +4,21 @@ title: Text Editors
 sidebar_label: Overview
 ---
 
+## Latest Metals server versions
+
+All editors use a separate Metals server underneath, which version can be
+changed idependently. It can be used to checkout newest features from the main
+branch.
```

Comment:
In theory though, this _could_ change moving forward no? Now that everything can be configured via `InitializationOptions`, there is no need to have a `metals-vscode` and `metals-emacs` but a single `metals` for each desired version.

Reply:
Hmm... right. MAybe then just remove the section.

Reply:
Moved the lower paragraph here.